test/Microsoft.Framework.Configuration.Binder.Test/ConfigurationBinderTests.cs

```diff
@@ -132,6 +133,36 @@ public void ShouldBeIgnoredTests(string property)
         }
 
         [Fact]
+        public void CanReadTimeSpanProperties()
+        {
+            var dic = new Dictionary<string, string>
+            {
+                {"TimeSpan", "356.23:59:59.999999"}
+            };
+            var builder = new ConfigurationBuilder(new MemoryConfigurationSource(dic));
+            var config = builder.Build();
+            var options = ConfigurationBinder.Bind<ComplexOptions>(config);
+            Assert.Equal(356, options.TimeSpan.Days);
+            Assert.Equal(999, options.TimeSpan.Milliseconds);
+        }
+        
+        [Fact]
+        public void CanReadDateTimeOffsetProperties()
+        {
+            var dic = new Dictionary<string, string>
+            {
+                {"DateTimeOffset", "05/22/2015 22:55:22 +02:00"}
+            };
+            var builder = new ConfigurationBuilder(new MemoryConfigurationSource(dic));
+            var config = builder.Build();
+            var options = ConfigurationBinder.Bind<ComplexOptions>(config);
+            Assert.Equal(2, options.DateTimeOffset.Offset.Hours);
+            Assert.Equal(5, options.DateTimeOffset.Month);
+            Assert.Equal(22, options.DateTimeOffset.Day);
+            Assert.Equal(2015, options.DateTimeOffset.Year);
+        }
+
+        [Fact]
```

Comment:
Might be worth adding a failure/parse issue for both of these to ensure stuff doesn't boom too badly